app/models/item.rb

```diff
@@ -0,0 +1,19 @@
+class Item < ApplicationRecord
+  #validates :name, presence: true,length: { minimum: 5 ,
+#too_short: "must have at least %{count} characters"}
+  validates :name,presence: true,length: { is: 8 },uniqueness: {case_sensitive: true}
+  validates :price, presence:true
+  validates :weight,numericality:  { greater_than: 100 }
+  #validates :in_stock,inclusion: { in: [true, false],message: 'this one is not allowed' }
```

Comment:
@MadhaviBellam  try not to place this commented code, make it as a clean code before pushing the code. In your next push, try to remove commented code.